scripts/seurat_obj_filter_final.Rmd

```diff
@@ -0,0 +1,121 @@
+---
```

Comment:
Yuo can keep this around if you want, but Dan has implemented this as a normal Rscript